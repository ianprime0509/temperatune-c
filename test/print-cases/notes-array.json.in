{
  "name": "Notes aren't an array",
  "octaveBaseName": "C",
  "referencePitch": 440,
  "referenceName": "A",
  "referenceOctave": 4,
  "notes": []
}
