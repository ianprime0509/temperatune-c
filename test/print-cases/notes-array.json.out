temperatune: notes must be an object
